docs/release-notes/8.0.asciidoc

```diff
@@ -5,6 +5,11 @@
 [[release-notes-8.0.0]]
 === 8.0.0
 
+[discrete]
+[[upgrade-reqs-8.0.0]]
+==== Upgrade requirements
+Before you upgrade, review the <<breaking-changes-8.0.0, breaking changes>> for this release and the <<upgrade-intro, {elastic-sec} upgrade instructions>>.
```

Comment:
How about calling the second linked page "upgrade guidelines" or something similar? Some of the content on that page is traditional instructions, and some of it is a list of considerations.
```suggestion
Before you upgrade, review the <<breaking-changes-8.0.0, breaking changes>> for this release and the <<upgrade-intro, {elastic-sec} upgrade guidelines>>.
```